pandas/tests/io/test_parquet.py

```diff
@@ -482,14 +502,15 @@ def test_categorical(self, fp):
         df = pd.DataFrame({'a': pd.Categorical(list('abc'))})
         check_round_trip(df, fp)
 
-    def test_datetime_tz(self, fp):
-        # doesn't preserve tz
```

Comment:
same need 2 tests (1 with each fixture)

Reply:
pyarrow is using similar style. there are already two tests for datetime_tz.